R/knit_print.R

```diff
@@ -21,6 +21,8 @@ knit_print.data.frame <- function(x, ...) {
     if (identical(context$df_print, knitr::kable)) {
       res <- one_string(c('<div class="kable-table">', '', knitr::kable(x), '', '</div>'))
       knitr::asis_output(res)
+    } else if (identical(context$df_print, print)) {
```

Comment:
@atusy, I understand this as if `context$df_print` is identical to `print` call `print(x)` else call `context$df_print(x)`. but in this case isn't it the same ?  `context$df_print(x)` will be the same as `print(x)`. What am I missing ?

Reply:
It should be the same, but happened to be different.
Anyway, I'll check the way to reproduce the issue.
Thanks!!